src/Controller/ContactController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\InfoCoach;
+use App\Form\ContactType;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ContactController extends AbstractController
+{
+    /**
+     * @Route("/contact/", name="app_contact")
+     * @return Response
+     */
+    public function index(Request $request, MailerInterface $mailer): Response
+    {
+        $form = $this->createForm(ContactType::class);
+        $form->handleRequest($request);
+        $coachInfo = $this->getDoctrine()
+            ->getRepository(InfoCoach::class)
+            ->findOneBy([]);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $dataForm = $_POST['contact'];
```

Comment:
erk, non ! pas de $_POST dans symfony, never !
tu devrais
- creer une entity Contact (sans annotation @ORM car pas relié à une BDD)
du coup tu fais un $contact = new Contact; 
et 
          $form = $this->createForm(ContactType::class, $contact);

puis ensuite si ton form isSubmitted tu as ton $contact hydraté avec les données du form !